view/frontend/web/js/model/checkout.js

```diff
@@ -272,20 +272,17 @@ function(
 
   function updateHasDeliveryOptions() {
     var isAllowed = false;
+    var shippingCountry = quote.shippingAddress().countryId;
 
-    Model.allowedShippingMethods().forEach(function(methodCode) {
-      var rate = Model.findRateByMethodCode(methodCode);
-      var shippingCountry = quote.shippingAddress().countryId;
+    if ('NL' === shippingCountry || 'BE' === shippingCountry) {
+      Model.allowedShippingMethods().forEach(function(methodCode) {
+        var rate = Model.findRateByMethodCode(methodCode);
 
-      if (rate && rate.available) {
-        isAllowed = true;
-      }
-
-      /* Only for MyParcelNL */
-      if (shippingCountry !== 'NL' && shippingCountry !== 'BE') {
-        isAllowed = false;
-      }
-    });
+        if (rate && rate.available) {
```

Comment:
moet dit niet ook weer op false? kan je niet dit doen:

```js
isAllowed = rate && rate.available;
```

Reply:
Ik denk dat dit niet meer op false moet wanneer eenmaal true, ik heb dit alleen gerefactored met gelijkblijvende functionaliteit, om niet de check op de country en het ophalen van het shippingadres in de foreach te hebben. Ik kan het ook terugzetten (met de country check in de foreach) maar dat leek me een waste of cpu cycles.

Reply:
ok... als het werkt 🤷🏼‍♀️